docs/usage/controls.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Controls
+order: 4
+---
+
+# Form controls
+
+Controls as we use the term here refer to the UI widgets that allow a user to enter data. In its most basic form that would be an `<input>`.
+
+ember-headless-form comes with support for the following controls built-in (but you can also use [custom controls](./custom-controls/index.md)), all yielded from the `Field` component:
```

Comment:
So weird! This link `[custom controls](./custom-controls/index.md)` works as expected but the above do not! 🤔